pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -266,7 +209,6 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 	reqChannel := make(chan *xdsapi.DiscoveryRequest, 1)
 	go receiveThread(con, reqChannel, &receiveError)
 
-	node := &core.Node{}
```

Comment:
@kyessenov please take a look I'm not sure you if specifically used this rather than con.node for a reason?

Reply:
This is a legacy var, @kyessenov just modify based on it.